### DISCUSSION TITLE
Next steps for festival

### DISCUSSION THREAD

**Discussion by julianpistorius** [resbazaz/festival2024#3] [General]
Hi @resbazaz/festival2024! 

We had our kickoff meeting on 2024-01-30. Check out [the notes](https://hackmd.io/@julianp-codecove/rkVA0lD96) for decisions made, and who volunteered for what.

Now that we have decided on 'hybrid' as [format of the 2024 festival](https://github.com/resbazaz/festival2024/discussions/2), we should decide on a date.

I created a first draft of a Google Form: https://forms.gle/DJT3WKgB8P6wPnsC9

Any feedback? Are we OK with using Google Forms? @lwdozal you're overseeing communications, so please chime in.

(Note: The Google Form doesn't currently check if a person actually controls the email address they enter, only that it is a correctly formatted address.)

**Comment by julianpistorius** [comment]
@brandon-jernigan & @val-pf I know you had concerns about two days being too short.

**Reply by brandon-jernigan** [reply]
It might be good to set up another meeting to discuss before we proceed with decisions. There have been a lot of topics and possibilities floating around such as outside, unconference, multi/single site, duration, etc.

**Reply by julianpistorius** [reply]
Problem: We want to host a festival that...

1. Accommodates the needs and preferences of our community members
2. Is manageable (and fun) for the organizers & volunteers to organize

Since key people are traveling at the moment, and in the spirit of making decisions through ['lazy consensus'](https://github.com/resbazaz/organization/blob/main/GOVERNANCE.md#voting-membership), I propose the following plan.

Use a mechanism like:

A. Public GitHub polls
B: Slack votes or polls
C: A Google Form (publishing a real-time summary of the responses)

For the ResBazAZ community to weigh in on the following 5 options:

**1. Multi vs single site**

Ask for interest in attending, as well as willingness to organize an in-person event wherever they are.

Options:

- Flagstaff
- Phoenix
- Tucson
- Other

If there is nobody willing to organize an in-person event in an area, then that answers that question.

**2. Unconference**  

Explain what it is, link to examples, compare it to the alternatives, with pros & cons.

**3. Outside vs inside**

Photos to give people an idea of what it is like.

Mention that outside in May is probably only feasible in Flagstaff, so consider that when they choose a week.

**4. Date**  

- Week of April 22nd
- Week of May 13th

**5. Duration**

Possibly also whether they prefer weekends, weekdays, or a combination.

---

Wait a reasonable amount of time (a week?).

Based on the community's responses, create a poll for the organizers and volunteers (hopefully now swollen in ranks and wider in geographical distribution) to indicate what they're able & willing to commit to.

Thoughts?

**Reply by e-eight** [reply]
I am fine with using either Github polls / Slack votes / Google Forms. Just prefer that most of the communication happens via a single medium, and not spread over different media.

**Comment by lwdozal** [comment]
I think google forms works! My only hesitation is sending out too many of them. But I'm assuming after this we won't send one out for a while, those being for volunteers or speaker submissions?

**Comment by jcoliver** [comment]
I slightly prefer Public GitHub polls. Tough to articulate why, I just do. :shrug:

**Comment by brandon-jernigan** [comment]
It might be good for the organizers here to talk about some of our ideas about what this might look like. For example:

Unconference: Can we discuss what an unconference style festival would look like? Are there particular activities that would be appealing for this? Would we want a mixture of unconference and conference style? From what I know this style can be a good way to focus more on community building by turning the "audience" in on itself.

Outdoors: There was discussion previously on slack about outside, and Julian posted pictures from the first ResBaz which included tents outside, with some advantages being that the festival is more visibility and discoverability. This would likely require a date earlier in April however. An outdoor day is a good match for unconference style, due to reduced need for screens and sunglare.

Duration: Will a low number of days limit the number of speakers to only a few? How many potential speakers do we think will want to participate? Will Hacky Hour act as the end of the festival, if so that reduces the amount of time for other things on that day.

Other Locations: Is anyone from those locations involved currently, is there appetite for organizers to do that?

Polls: Agreed on limiting the number of polls, but might also be good to put together only a few configurations of options that organizers think are coherent, work well together and that we have the stomach to work toward.

**Reply by jcoliver** [reply]
Re: Locations. I'm in the Library and can say that the Library would definitely support ResBaz, if the indoor space fits the needs. We would have to check on availability of spaces, but it is certainly an option.

**Reply by julianpistorius** [reply]
Thank you @jcoliver & @brandon-jernigan!

Any ideas from other folks in @resbazaz/festival2024 ? Especially @egreich & @rbelshe.

**Reply by julianpistorius** [reply]
Regarding unconference see: https://en.wikipedia.org/wiki/Unconference

Some of the mentioned facilitation styles sound like fun:

> - [Birds of a feather](https://en.wiktionary.org/wiki/birds-of-a-feather_session)
> - [Dotmocracy](https://en.wikipedia.org/wiki/Dotmocracy)
> - [Fishbowl](https://en.wikipedia.org/wiki/Fishbowl_(conversation))
> - [Ignite](https://en.wikipedia.org/wiki/Ignite_(event))
> - [Lightning talks](https://en.wikipedia.org/wiki/Lightning_talk)
> - [Open Space Technology](https://en.wikipedia.org/wiki/Open_Space_Technology)
> - [PechaKucha](https://en.wikipedia.org/wiki/PechaKucha)
> - [Speed geeking](https://en.wikipedia.org/wiki/Speed_geeking)
> - [World café (conversation)](https://en.wikipedia.org/wiki/World_caf%C3%A9_(conversation))

A helpful guide here: [Ten Simple Rules for Organizing an Unconference](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4310607/)

**Reply by hidyverse** [reply]
> Especially @egreich & @rbelshe.

@julianpistorius Have we tried emailing?

**Reply by julianpistorius** [reply]
Not yet.

**Reply by hidyverse** [reply]
Emailed!

**Comment by hidyverse** [comment]
I don't agree we need to poll further (given historically our results haven't been meaningful). I would, rather, try to accommodate all the ideas included here, as these are the voices of our conference attendees. Any choice we make will be perfect, but the number of choices here is a bit daunting for conversing in large groups.  

What about 3 days of ResBaz conference: 1 IRL unconference day, 1 IRL conference day, 1 virtual conference day (could be together with other festivals if organizers decide). 
For those IRL days, what about setting up inside _and_ outside? 

Finally, I think we can be better community members to outside-of-Tucson festivals if we organize and document our conference journey (but also our ResBaz conference mission).

**Comment by hidyverse** [comment]
Great conversations, all. 
I have created another (hopefully final) poll (#6) to come to consensus on our 2024 event format.
Please continue necessary conversations with your vote in the poll.

**Reply by lwdozal** [reply]
Thank you! This is helpful. I just voted.

**Reply by julianpistorius** [reply]
Yes, thank you @hidyverse! Also voted.
